cal/cal/models.py

```diff
@@ -583,20 +584,38 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         return events_qs
 
-    def get_hours_per_week(self, calendar_ids=None, start=None, end=None):
+    def get_hours_per_week(self, timezone, calendar_ids=None, start=None, end=None):
         """
         Returns a list of week-hour tuples corresponding to the events in this ColorCategory.
         Each week starts at the start time.
         """
         week_hours = []
         events = self.query().order_by('start')
         i = 0
-        start = events[0].start
+        # Convert start to local time
+        start = handle_time_string(str(events[0].start), timezone)
+        # Change start date to be Monday beginning of week
+        while start.weekday() != 0:
+            start = start - relativedelta(days=1)
```

Comment:
You should use datetime.timedelta because we're already using that everywhere else in our codebase. I don't think there is any place where we need to use dateutil because the django time utilities + python datetime + pytz are more than enough for everything we need, and we should try to stay consistent with our libraries if we can.